iroha/src/lib.rs

```diff
@@ -35,6 +35,9 @@ use async_std::{
 };
 use std::{path::Path, sync::Arc, time::Duration};
 
+/// The interval at which sumeragi checks if there are tx in the `queue`.
+pub const TX_RETRIEVAL_INTERVAL: Duration = Duration::from_millis(100);
```

Comment:
I thought that we should use block build step ms for this purpose?

Reply:
If the peer is a leader then, using block build time would be reasonable. But this is not always the case. Most of the time the peer is not a leader, so it just forwards the tx it received to the leader. Taking into account this case, waiting for block build time, will be misleading.

Reply:
Ok - can you create the follow up issue to catch on this and create additional configuration parameter?